packages/valory/skills/hello_world_abci/rounds.py

```diff
@@ -91,7 +91,7 @@ def end_block(self) -> Optional[Tuple[BaseSynchronizedData, Event]]:
 
         if self.collection_threshold_reached:
             synchronized_data = self.synchronized_data.update(
-                participants=tuple(self.collection),
+                participants=sorted(tuple(self.collection)),
```

Comment:
tuple is immutable. So you're creating a list again here I think. Swap it around
```suggestion
                participants=tuple(sorted(self.collection)),
```